jsapp/js/components/formLanding.es6

```diff
@@ -129,8 +134,31 @@ export class FormLanding extends React.Component {
       asset: this.state
     });
   }
+  increaseDVCOUNT_LIMIT() {
+    if (this.state.DVCOUNT_LIMIT + DVCOUNT_LIMIT_MINIMUM <= this.state.deployed_versions.count + DVCOUNT_LIMIT_MINIMUM) {
+      this.setState({ DVCOUNT_LIMIT : this.state.DVCOUNT_LIMIT + DVCOUNT_LIMIT_MINIMUM });
+    }
+    if (this.state.DVCOUNT_LIMIT >= 80) {
+      var curr = this.state.currentVersions;
+      if (this.state.nextPageUrl) {
```

Comment:
To avoid setting `nextPageUrl` in render cycle, let's do it this way:

```javascript
let urlToLoad = null;
if(this.state.furtherPageUrl) {
  urlToLoad = this.state.furtherPageUrl;
} else if (this.state.deployed_versions.next) {
  urlToLoad = this.state.deployed_versions.next;
}
if (urlToLoad !== null) {
  dataInterface.loadNextPageUrl(urlToLoad).done((data) => {
    this.setState({furtherPageUrl: data.deployed_versions.next});
    Object.values(data.deployed_versions.results).forEach(function(ele){
        curr.push(ele);
    });
  });
}
```
This way you will either load first `.next` or second `.next` and further ones :)